client/src/components/Login/Login.js

```diff
@@ -88,65 +82,86 @@ export default function SignInSide() {
-            >
-              Sign In
-            </Button>
-            <Grid container>
-              <Grid item xs>
-                <Link href="#" variant="body2">
-                  Forgot password?
-                </Link>
-              </Grid>
-              <Grid item>
-                <Link href="#" variant="body2">
-                  {"Don't have an account? Sign Up"}
-                </Link>
-              </Grid>
+    <div>
+    <Grid container>
+        <Grid
+            item xs={4}
+            direction='row'
+        >
+            <UnauthenticatedSidebar/>
+        </Grid>
+        <Grid
+            item xs={8}
+            direction='row' >
+              
+          <Container  className={classes.mainContainer} component="main" maxWidth="xs">
+            <CssBaseline />
+            <Grid container justify='flex-end'>
+                  <Grid item>
+                    <Typography classes = {{ body2: classes.body2Style }} component="p" variant="body2">
+                      Don't have an account? 
+                      <Link className={classes.routerLink} to='/signup'>
+                      <Button color='primary' classes = {{ root: classes.buttonMarginRight }} variant="outlined" size="large">
+                          Create Account
+                      </Button>
+                      </Link>
+                      </Typography>                     
+                  </Grid>
             </Grid>
-            <Box mt={5}>
-              <Copyright />
-            </Box>
-          </form>
-        </div>
-      </Grid>
+            <div className={classes.paper}>
+              <Typography className={classes.welcomeHeading} component="h2" variant="h4">
+                Welcome Back!
+              </Typography>
+              <form className={classes.form} noValidate>
+                <Grid container spacing={2}>
+                  <Grid item xs={12}>
+                    <TextField
+                      required
+                      fullWidth
+                      id="email"
+                      label="Email Address"
+                      name="email"
+                      autoComplete="email"
+                      InputProps = {{ 
+                        classes: {root: classes.textFieldSelectLabel} 
+                      }}
+                      InputLabelProps = {{ 
+                        classes: {root: classes.textFieldSelectLabel} 
+                      }}
+                    />
+                  </Grid>
+                  <Grid item xs={12}>
+                    <TextField
+                      required
+                      fullWidth
+                      name="password"
+                      label="Password"
+                      type="password"
+                      id="password"
+                      autoComplete="current-password"
+                      InputProps = {{ 
+                        endAdornment: <InputAdornment position='end'> Forgot? </InputAdornment>,
```

Comment:
If you want to change this color, there's an issue with giving InputAdornment straight up text. You need to give it <Typography className={classes.adornmentStyle}>Forgot?</Typography> where "adornmentStyle" contains the colour you want. 
This is because in the implementation of InputAdornment ([here](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/InputAdornment/InputAdornment.js#L91)), if you give it text, it creates a Typography by itself and gives it "textSecondary" as the colour.

Reply:
Thanks for this approach Sina! Works like a charm